packages/server/src/notification/notification.controller.ts

```diff
@@ -1,19 +1,30 @@
-import { Body, Controller, Get, Param, Post, UseGuards } from '@nestjs/common';
+import {
+  Body,
+  Controller,
+  Get,
+  Param,
+  Post,
+  Req,
+  Res,
+  UseGuards,
+} from '@nestjs/common';
 import { DesktopNotifBody } from '@template/common';
+import { Request, Response } from 'express';
 import { JwtAuthGuard } from '../profile/jwt-auth.guard';
 import { NotificationService } from './notification.service';
 
 @Controller('notifications')
-@UseGuards(JwtAuthGuard)
 export class NotificationController {
   constructor(private notifService: NotificationService) {}
 
   @Get('desktop/credentials')
+  @UseGuards(JwtAuthGuard)
```

Comment:
this is so there's no guard on the verify endpoint , so twilio can actually interact with our API